app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss

```diff
@@ -104,6 +104,12 @@ $padding-small-horizontal: 3px;
   text-align: left;
 }
 
+.user_info-logged_in-account_dropdown-storage-name {
```

Comment:
why does this class name forego the .navigation- module convention?

Reply:
I was following the class name convention of the parent element

<img width="716" alt="screen shot 2018-02-05 at 10 20 25 am" src="https://user-images.githubusercontent.com/19269161/35812183-4b5ab91c-0a5e-11e8-8058-69d51282347d.png">